core/src/Data/Query/KeyQuery.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Harmony\Core\Repository\Query;
+namespace Harmony\Core\Data\Query;
 
 class KeyQuery implements Query {
```

Comment:
Why keeping this class if we removed it from the InMemoryDataSource, which is the context where this is used? I would revert the changes in the InMemory. This is our Fundamentals:
https://harmony.mobilejazz.com/docs/fundamentals/data/data-source/query#keyquery-support

Reply:
Fixed.